.travis.yml

```
@@ -10,11 +15,16 @@ python:
  - "pypy"
  - "pypy3"
 install:
+ - ./deps.sh --yes
  - pip install coverage
- - pip install backports.lzma
  - ./setup.py install
 script: ./setup.py test
 cache: pip
 matrix:
```

```suggestion
jobs:
```
Travis CI has replaced `matrix` with `jobs`.

---

Changed.